app/forms/Mappings.scala

```diff
@@ -159,6 +159,10 @@ object Mappings {
       .verifying(emptyErrorKey, paramVal => paramVal == true.toString || paramVal == false.toString)
   }
 
+  def nonemptyBoolean(emptyErrorKey: String): Mapping[Boolean] = {
+    boolean verifying (emptyErrorKey, value => value == true || value == false)
```

Comment:
a bit confusing. `value` is a boolean type, so you can write: `value => value || !value` but anyway it will always give true - `value => true`